jest.config.js

```diff
@@ -0,0 +1,15 @@
+/** @type {import('ts-jest').JestConfigWithTsJest} */
+export default {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  transform: {
+    '^.+\\.ts?$': 'ts-jest',
+    "^.+\\.[t|j]sx?$": "babel-jest"
+  },
+  transformIgnorePatterns: ["node_modules/(?!isomorphic-unfetch)"],
+  automock: false,
+  resetMocks: false,
+  setupFiles: [
+    "./jestSetup.ts"
+  ]
```

Comment:
Have you considered setting `collectCoverage: true`? If you do decide to, can I recommend using the npm `c8` package.